src/query/interpret/eval.hpp

```diff
@@ -1121,11 +1121,11 @@ class ExpressionEvaluator : public ExpressionVisitor<TypedValue> {
           throw QueryRuntimeException("Unexpected error when getting properties.");
       }
     }
-    return *maybe_props;
+    return *std::move(maybe_props);
```

Comment:
Isn't it better to leave so that RVO kicks in? Can it kick in here?

Reply:
Nope, no RVO does not apply here